### ISSUE TITLE
[Assistive tech and device support] UI orientation doesn't support both portrait and landscape mode. (11.01.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#29340]
#### VFS team name
Education Application (BAH)

#### VFS product name
Comparison Tool Redesign

#### Point of Contact/Reviewers
Trevor Pierce (Accessibility)

---

### Platform Issue
UI orientation doesn't support both portrait and landscape mode.

### Issue Details
Can't tap on the black map icons in landscape like I can in portrait. This behavior must be consistent across orientation.

### Link, screenshot or steps to recreate
![Screen Shot 2021-08-26 at 5 06 08 PM](https://user-images.githubusercontent.com/934879/131042367-b279753c-d409-4347-8f4d-01b9d3a324c9.png)

### VA.gov Experience Standard
[Cateogy Number 11, Issue Number 01](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.4_AA

---

### Platform Recommendation
Make the map behavior consistent on tap/click events.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by zurbergram** [comment]
@shiragoodman It looks like based on your image the tapping on the markers is working as intended indicated by the focus box around a result above the map


https://user-images.githubusercontent.com/1094999/132355073-63d20caa-0423-4047-8808-8b96be05b4e2.mov

**Comment by zurbergram** [comment]
@1Copenut It looks like based on your image the tapping on the markers is working as intended indicated by the focus box around a result above the map

Notice in video that results is scrolling to corresponding marker above the map

https://user-images.githubusercontent.com/1094999/132355073-63d20caa-0423-4047-8808-8b96be05b4e2.mov

**Comment by zurbergram** [comment]
@noahgelman

**Comment by af-manish** [comment]
@noahgelman The link has since been updated. Please provide us your thoughts on the above:
https://staging.va.gov/education/gi-bill-comparison-tool/?search=location&name=chicago&location=dc

**Comment by mrblanco1** [comment]
When clicking on a location the search engine is unresponsive (no results are provided)

**Comment by AdamFreemer** [comment]
After a call with @hallihocine , there seem to be 2 defined problems:
1) In landscape view, when tapping on an Institution in landscape mobile view, the view should be switched to list, like in portrait mode.
2) Institutions located vertically where the Search button is aren't clickable, because of some interference with div `search-area-control-container`. These Institutions should be tappable. 

@daltonatkins @mrblanco1 just confirming.

**Comment by mrblanco1** [comment]
> After a call with @hallihocine , there seem to be 2 defined problems:
> 
> 1. In landscape view, when tapping on an Institution in landscape mobile view, the view should be switched to list, like in portrait mode.
> 2. Institutions located vertically where the Search button is aren't clickable, because of some interference with div `search-area-control-container`. These Institutions should be tappable.
> 
> @daltonatkins @mrblanco1 just confirming.

Correct. Please go ahead and focus on the above listed tasks.  Thanks!

**Comment by AdamFreemer** [comment]
After some review of current code, I think the best way to proceed is to enable the List / Map tabs for the portrait view by expanding the mobile portrait view to include landscape too. I am halting working on this card to proceed with https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/36247 which I started previously and is now unblocked.

**Comment by hallihocine** [comment]
Ticket has been validated in staging: I can tap/click on the black map icons in landscape mode. The UI orientation does support both portrait and landscape mode and the behavior is consistent across orientation. See below: 

https://user-images.githubusercontent.com/86266024/157108598-2b99ca0e-0059-4ed6-a1e7-0ed64280bdeb.mp4

**Comment by hallihocine** [comment]
- Test Run in Test Rail: 
https://dsvavsp.testrail.io/index.php?/runs/view/3029&group_by=cases:section_id&group_order=asc&group_id=3710
- Test step:
[UI orientation doesn't support both portrait and landscape mode.xlsx](https://github.com/department-of-veterans-affairs/va.gov-team/files/8200499/UI.orientation.doesn.t.support.both.portrait.and.landscape.mode.xlsx)
